resources/views/docs/1/digging-deeper/child-process.md

```diff
@@ -302,18 +262,53 @@ Alternatively, you may use the `ChildProcess` facade to restart a process via it
 ChildProcess::restart('tail');
 ```
 
+## Handling output
+
+A Child Process may send output via any of the following interfaces:
+
+-   Its standard output stream (`STDOUT`).
+-   Its standard error stream (`STDERR`).
+-   A custom interface, e.g. a network socket.
+-   Broadcasting a Custom Event
+
+`STDOUT`, `STDERR` & [Custom Events](/docs/1/digging-deeper/broadcasting#custom-events) are dispatched using Laravel's event system.
+
+You may listen to these events by registering a listener in your app service provider, or on the front end using the [Native helper](/docs/1/digging-deeper/broadcasting#listening-with-javascript).
+
+Please see the [Events](#events) section for a full list of events.
+
+### Listening for Output (`STDOUT`)
+
+You may receive standard output for a process by registering an event listener for the
+[`MessageReceived`](#codemessagereceivedcode) event:
+
+### Listening for Errors (`STDERR`)
+
+You may receive standard errors for a process by registering an event listener for the
+[`ErrorReceived`](#codeerrorreceivedcode) event:
+
 ## Events
 
 NativePHP provides a simple way to listen for Child Process events.
 
-All events get dispatched as regular Laravel events, so you may use your `EventServiceProvider` to register listeners.
+All events get dispatched as regular Laravel events, so you may use your `AppServiceProvider` to register listeners.
 
 ```php
-protected $listen = [
-    'Native\Laravel\Events\ChildProcess\MessageReceived' => [
-        'App\Listeners\MainLoopEvent',
-    ],
-];
+use Illuminate\Support\Facades\Event;
+use Native\Laravel\Events\ChildProcess\MessageReceived;
+
+/**
+ * Bootstrap any application services.
+ */
+public function boot(): void
+{
+    Event::listen(MessageReceived::class, function(MessageReceived $event) {
+        if ($event->alias === 'tail') {
+            //
+        }
+    });
+}
+
```

Comment:
I removed the `protected $listen = ` example in favour for `Event::listen` in the app service provider.

Since Laravel 11 the property approach was removed from the docs. Having two examples that do the same thing right next to each other seemed confusing to me